docs/deploying.rst

```diff
@@ -82,6 +82,20 @@ A requirements.txt file will be created out of the ``Pipfile`` so like the
 requirements file above, it should not contain packages that are already part
 of the stack.
 
+If you use `poetry`_ you can specify your ``pyproject.toml``::
+
+    # project_directory/scrapinghub.yml
+
+    projects:
+      default: 12345
+      prod: 33333
+
+    requirements:
+      file: pyproject.toml
+
+A ``poetry.lock`` file must be available, that will be used for determining the
+full requirements.
```

Comment:
As a suggestion: recently I got a request for improving the deploy docs to mention a shortcut to install pipenv  (which was addressed by https://github.com/scrapinghub/shub/commit/f9d31eb9013433e88837d4039853c549de7fef5f), I wonder if it makes sense to do similar thing here, wdyt?

Reply:
Yeah, sure!  I'd assume if they want to use poetry.lock they're already aware of what Poetry is, its official website, etc.  Maybe that request was from the perspective of someone being introduced to a project that uses pipenv but who hasn't previously used the tool?  What about a link to the website, like "\[Poetry](x) is a Python dependency and virtual management tool"

Reply:
> Maybe that request was from the perspective of someone being introduced to a project that uses pipenv but who hasn't previously used the tool? What about a link to the website, like "[Poetry](x) is a Python dependency and virtual management tool"

Yeah, I think so. Sounds good, I'd go with [the official definition](https://poetry.eustace.io/docs/): `is a tool for dependency management and packaging in Python`.

Reply:
Also, there's a little dilemma here: from one side, we're trying to keep requirements.txt minimal and avoid shadowing python dependencies already installed in stack (to avoid side effects, speed up deployments etc), so the same rules should be applied to pipfile/poetry (lock) files. From the other side, I assume that pipfile/poetry are used by devs to setup a full environment from scratch, thus it has almost all the deps already installed in stack plus additional ones. And having different sets of files for local development and for SC deployment doesn't sound right. For sure, it's out of the scope of the current PR, so feel free to ignore it for now, but it's something we'll have to think more about in the future.